e2e/rbd.go

```diff
@@ -427,6 +439,53 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("Resize rbd-nbd PVC and check application directory size", func() {
```

Comment:
can this be tested on the minikube? looks like minikube kernel version is still 4.19

Reply:
This is why we added a defence check here for the kernel version. So that when we have a new minikube version with kernel > 5.x this test goes smooth, else it will be skipped